<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="robotIdLabel">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>0</y>
      <width>66</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Robot ID</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="torsoProgressBar">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>90</y>
      <width>118</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QProgressBar" name="leftLegProgressBar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>118</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QProgressBar" name="rightLegProgressBar">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>200</y>
      <width>118</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QLabel" name="leftLegLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>91</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Левая нога</string>
    </property>
   </widget>
   <widget class="QLabel" name="roghtLegLabel">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>180</y>
      <width>91</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Правая нога</string>
    </property>
   </widget>
   <widget class="QLabel" name="torsoLabel">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>70</y>
      <width>66</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Торс</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="gunProgressBar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>118</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QProgressBar" name="grabManipProgressBar">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>140</y>
      <width>118</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QProgressBar" name="gunManipProgressBar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>118</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QLabel" name="gunManipLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>141</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Орудийный манип</string>
    </property>
   </widget>
   <widget class="QLabel" name="gunLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>66</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Орудие</string>
    </property>
   </widget>
   <widget class="QLabel" name="grabManipLabel">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>120</y>
      <width>101</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Хватат. манип</string>
    </property>
   </widget>
   <widget class="QLabel" name="repairStationLabel">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>70</y>
      <width>141</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Ремонтная станция</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="repairStationLabel_2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>90</y>
      <width>118</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QLabel" name="componentIdLabel">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>120</y>
      <width>111</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>ID компонента: </string>
    </property>
   </widget>
   <widget class="QComboBox" name="componentIdComboBox">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>120</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>50</y>
      <width>131</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QPushButton" name="getStateButton">
       <property name="text">
        <string>Состояние меха</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="repairButton">
       <property name="text">
        <string>Ремонт</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="shootButton">
       <property name="text">
        <string>Выстрел</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mapWidget">
       <property name="text">
        <string>Карта</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="numOfCart">
       <property name="text">
        <string>Кол-во патронов:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="gunAmmoLabel">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="moveForwardButton">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>210</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>^</string>
    </property>
   </widget>
   <widget class="QPushButton" name="moveLeftButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>220</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="moveRightButton">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>220</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="moveBackwardButton">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>230</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>v</string>
    </property>
   </widget>
   <widget class="QLabel" name="moveLabel">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>190</y>
      <width>91</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Движение</string>
    </property>
   </widget>
   <widget class="QLabel" name="turnLabel">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>190</y>
      <width>91</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Поворот</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rotateLeftButton">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>220</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rotateRightButton">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>220</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="CurrentPositionLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>151</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Текущее положение:</string>
    </property>
   </widget>
   <widget class="QLabel" name="xEdit">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>270</y>
      <width>21</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QLabel" name="yEdit">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>270</y>
      <width>21</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>y</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
